docs/guides/runbooks_steps_microsoft_teams.md

```diff
@@ -0,0 +1,222 @@
+    })
+  }
+
+  repeats   = false
+  automatic = true
+}
+```
+
+### Notify Channel With a Custom Message - Steps Argument Reference
+
+* `action_id` - (Required) The ID of the runbook action for the step.
+* `config` - (Required) JSON string representing the configuration settings for the step.
+  Use [Terraform's jsonencode](https://www.terraform.io/language/functions/jsonencode)
+  function so that [Terraform can guarantee valid JSON syntax](https://www.terraform.io/language/expressions/strings#generating-json-or-yaml).
+* `name` - (Required) The name of the step.
+* `automatic` - (Optional) Whether this step should automatically execute.
+* `repeats` - (Optional) Whether this step should repeat. Defaults to `false`.
+  When this value is `true`, `repeats_duration` _must_ be provided.
+* `repeats_duration` - (Optional) How often this step should repeat in ISO8601.
+  Example: PT10M [Format Spec](https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm)
+  This value _must_ be provided if `repeats` is `true`. This value _must not_ be provided if `repeats` is `false`.
+
+The `config` block supports:
+
+* `channels` - (Required) A comma separated list of channels to notify.
+* `message` - (Required) The custom message to send to the list of channels.
+  This field supports [FireHydrant's template variables](https://support.firehydrant.com/hc/en-us/articles/4409136426004-Using-template-variables-in-Runbooks)
+  so you can automatically include details such as when the incident started, the incident summary, severity, roles involved, and much more.
+
+## Notify Incident Channel With a Custom Message
+
+The [Microsoft Teams **Notify Incident Channel With a Custom Message** step](https://support.firehydrant.com/hc/en-us/articles/360058202811)
+allows FireHydrant users to automatically notify the incident channel with a custom message.
+
+### Notify Incident Channel With a Custom Message - Example Usage
+
+Basic usage:
+```hcl
+data "firehydrant_runbook_action" "microsoft_teams_notify_incident_channel_custom" {
+  integration_slug = "microsoft_teams"
+  slug             = "notify_incident_channel_custom_message"
+  type             = "incident"
+}
+
+resource "firehydrant_runbook" "microsoft_teams_notify_incident_channel_custom_runbook" {
+  name = "microsoft-teams-notify-incident-channel-custom-runbook"
+
+  steps {
+    name      = "Notify the Microsoft Teams Incident Channel with a Custom Message"
+    action_id = data.firehydrant_runbook_action.microsoft_teams_notify_incident_channel_custom.id
+
+    config = jsonencode({
+      message = "Please check-in with your current status on this {{ incident.severity }} incident\n\n```\n/firehydrant add note I'm calculating the power required by the flux capacitor\n```\n"
+    })
+  }
+  
+  automatic = true
```

Comment:
move automatic into step block
```suggestion
  steps {
    name      = "Notify the Microsoft Teams Incident Channel with a Custom Message"
    action_id = data.firehydrant_runbook_action.microsoft_teams_notify_incident_channel_custom.id

    config = jsonencode({
      message = "Please check-in with your current status on this {{ incident.severity }} incident\n\n```\n/firehydrant add note I'm calculating the power required by the flux capacitor\n```\n"
    })
    
    automatic = true
  }
  

```